helpers/base.py

```diff
@@ -217,3 +218,19 @@ def data_has_value_from_substring_list(data, needle_list):
         return False
 
     return any(needle in data for needle in needle_list)
+
+def safe_json_loads(data):
+    """Safely load a JSON string into a dictionary
+
+    Args:
+        data (str): A JSON string
+
+    Returns:
+        dict: The loaded JSON string or empty dict
+    """
+    try:
+        json_data = json.loads(data)
```

Comment:
you could just do the return here directly instead of in the else below:  `return json.loads(data)`